src/main/flight/pid.c

```diff
@@ -717,6 +737,12 @@ static void FAST_CODE pidApplyMulticopterRateController(pidState_t *pidState, fl
     float itermErrorRate = rateError;
     applyItermRelax(axis, pidState->gyroRate, pidState->rateTarget, &itermErrorRate);
 
+#ifdef USE_ANTIGRAVITY
```

Comment:
Antigravity should only be applied in ACRO (or at least in modes with manual throttling). For modes with automatic throttling it doesn't make much sense anyway.

Reply:
actually I do not think adding extra condition makes sense. If user controls the throttle the he does. When FC controls the throttle then either changes and soft and antigravity is not active or very rapid (enabling AltHol) and then antigravity does its thing